packages/atomic/src/components/common/atomic-modal/atomic-modal.tsx

```diff
@@ -53,6 +53,11 @@ export class AtomicModal implements InitializableComponent<AnyBindings> {
   @Prop({mutable: true}) container?: HTMLElement;
   @Prop({reflect: true, mutable: true}) isOpen = false;
   @Prop({mutable: true}) close: () => void = () => (this.isOpen = false);
+  /**
+   * Wether to display the open and close animations over the entire page or the atomic-modal only.
+   */
+  @Prop({reflect: true}) animateOverEntirePage = true;
+  @Prop({reflect: true}) isIPX = false;
```

Comment:
Could atomic-ipx-modal be its own component instead? I know the DX team is not super kean with forks in base components 🤔

Reply:
@btaillon

Reply:
I thought it already was? I guess you mean for the refine modal.

I don't think there's a significant enough difference between the two to need a new refine modal component. The biggest difference here is some styling/animation and the space occupied by the component. I am actually tempted to investigate this myself and see if I'm luckier in finding a solution that removes the need for the `isIPX` parameter. Since this is internal, it's not a problem for now.

Reply:
Ish ouin un bool comme ca c'est semi 😐

Reply:
Hmm yeah I'm a bit confused. Why do we need to change some props in `atomic-modal` if we don't directly use it in an IPX? 🤔

Reply:
we need to pass it to `refine-modal-common` so that it gets passed to `atomic-modal` so that we can override the animation settings :/

Reply:
Yeah that's the exact kind of thing that should not happening the project, this is a slippery slope. This needs to stay generic.

Just a very slight change that will make all the difference would be to have a prop on the animation name itself.

```
@Prop() animationOnOpen = 'animateScaleUpModal'
@Prop() animationOnClose = 'animateSlideDownModal'
```

And you can then leverage tailwind to pass arbitrary animation.
eg: https://tailwindcss.com/docs/animation#arbitrary-values